tests/queries/0_stateless/02815_alias_to_length.sql

```diff
@@ -0,0 +1,2 @@
+SELECT OCTET_LENGTH([1,2,3,4]);
+SELECT OCTET_LENGTH('1234');
```

Comment:
Let's add a test checking that it's case insensitive,
and also a test checking that it counts bytes instead of code points (for example, with Chinese characters).